modules/wri_zoom/wri_zoom.post_update.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Zoom module post_update implementations.
+ */
+
+/**
+ * Transfer Orto integration settings to new configuration option.
+ */
+function wri_zoom_post_update_default_admin_settings() {
+  $orto_url = \Drupal::config('content_snippets.content')->get('snippets')['orto_registration_url'];
+  if (!empty($orto_url)) {
+    $config_factory = \Drupal::configFactory();
+    $config = $config_factory->getEditable('wri_zoom.settings');
+    $config->set('orto_enabled', TRUE);
+    $config->set('orto_registration_url', $orto_url);
+    $config->save(TRUE);
```

Comment:
This is great. One final nitpick: Can you also remove the `orto_registration_url` snippet from the config after doing this? I suspect the same commands we used to install it would work?
```
  // Import the content_snippets:
  \Drupal::service('distro_helper.updates')->updateConfig('content_snippets.content', ['snippets#orto_registration_url'], 'wri_listing_block');
  \Drupal::service('distro_helper.updates')->updateConfig('content_snippets.items', [
    'orto_registration_url',
  ], 'wri_admin');
```

I don't want people to be confused by that content snippet still existing.